core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -615,81 +653,75 @@ var ExplorationEditorMainTab = function() {
     }
   };
 
+  /**
+   * Parse for Angular selectors and remove them.
+   * @param {string} [ruleDescription] - Interaction type.
+   * @param {string|string[]} [providedText] - Feedback text to replace with.
+   */
+  var _replaceAngularSelectors = function(ruleDescription, providedText) {
+    // Expected input = is equal to {{a|NonnegativeInt}} and
```

Comment:
I don't understand the significance of this comment. Are you trying to provide an example? If not then why are we only dealing with NonnegativeInt here?

Reply:
I was just trying to provide an example. The ruleDescription for various interaction are from [rule_template.json](https://github.com/oppia/oppia/blob/develop/extensions/interactions/rule_templates.json).

Reply:
Ah ok. Maybe amend the docstring to make it clear that it's an example?

It's fine to be verbose in docstrings, just make sure to be really clear.